src/validation/emailValidation.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/prefer-default-export */
+import { regexForEmail } from '../__mocks__/emailVerification';
+
+export const emailRegex = (email) => {
+  // eslint-disable-next-line no-useless-escape
```

Comment:
why this disable `no-useless-escape`

Reply:
It is not a useless escape as it would affect my regex if i remove the escape character. The escape character makes it possible to be able to write more regex.